core/src/com/testing/objects/RectangleButton.java

```diff
@@ -0,0 +1,96 @@
+package com.testing.objects;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+import com.testing.oti.OTIDrawable;
+
+public class RectangleButton extends Button implements OTIDrawable {
```

Comment:
textures would be preferred over a shape renderer

Reply:
The reason a shape renderer is used is because it allows you to generate color for any number of elements. also textures don't need to be stored if there are no textures.

Reply:
in that case, the name `RectangleButton` should probably be changed to something like "ColourButton"?

Reply:
Done